game-core/src/main/java/games/strategy/engine/framework/startup/ui/posted/game/pbem/EmailSenderEditorViewModel.java

```diff
@@ -100,10 +104,18 @@ boolean showEmailOptions() {
     return !isEmailProviderDisabled();
   }
 
-  private boolean isEmailProviderDisabled() {
+  protected boolean isEmailProviderDisabled() {
     return selectedProvider.equals(PROVIDER_DISABLED);
   }
 
+  protected boolean isEmailProviderGmail() {
+    return selectedProvider.equals(EmailProviderPreset.GMAIL.getName());
+  }
+
+  protected boolean isEmailProviderHotmail() {
```

Comment:
These two methods use in combination with the `isEmailProviderDisabled` is where I see a problem. Combining these three into one method I would suggest is the way to go to avoid having any logic in the view and to fit very cleanly with the existing pattern.

Reply:
Updated to a single method.